javascript/javascript3/week3/screenshot_app.js

```diff
@@ -0,0 +1,121 @@
+const BASE_URL = "https://crudcrud.com/api/";
+const urlInput = document.getElementById("urlInput");
+const generateAndSaveScreenshotBtn =
+  document.getElementById("getScreenshotBtn");
+const savingMsg = document.getElementById("savingMsg");
+const displayListBtn = document.getElementById("showBtn");
+const savedImgsEl = document.getElementById("savedImgs");
+
+let screenshotURL;
+
+const generateScreenshot = async () => {
+  if (urlInput.value === "") {
```

Comment:
if(urlInput.value) would achieve the same thing since empty string are counted as being falsy

Reply:
Hi nowalor, 
here did you mean (!urlinpt.value)?
Because if I use (urlinput.value) I get an alert of invalid URL.